electrochemicalquantities.ttl

```diff
@@ -437,15 +445,34 @@ and t* through mathematical models, provided that the long-time potential- deter
 [ rdf:type owl:Axiom ;
    owl:annotatedSource :electrochemistry_264f40d1_17c9_4bc7_9c47_5cfb18132278 ;
    owl:annotatedProperty emmo:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 ;
-   owl:annotatedTarget "The rate of change of applied potential with time."@en ;
+   owl:annotatedTarget "the rate of change of applied potential with time"@en ;
    dcterms:source "J. M. Pingarrón et al., Terminology of electrochemical methods of analysis (IUPAC Recommendations 2019), Pure and Applied Chemistry, 4, 92, 2020, 641-694. https://doi.org/10.1515/pac-2018-0109"
  ] .
 
 
+###  http://emmo.info/electrochemistry#electrochemistry_265bb4d6_5eec_40f6_a3fa_59b3bd08e9af
+:electrochemistry_265bb4d6_5eec_40f6_a3fa_59b3bd08e9af rdf:type owl:Class ;
+                                                       rdfs:subClassOf :electrochemistry_aecc6094_c6a5_4a36_a825_8a497a2ae112 ,
+                                                                       emmo:EMMO_4f2d3939_91b1_4001_b8ab_7d19074bf845 ;
+                                                       emmo:EMMO_50c298c2_55a2_4068_b3ac_4e948c33181f "https://www.electropedia.org/iev/iev.nsf/display?openform&ievref=114-03-12" ;
+                                                       emmo:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "sum of the absolute values of the potential differences resulting from anodic and cathodic polarizations of an electrochemical cell"@en ;
+                                                       skos:altLabel "CellPolarizationPotential"@en ;
+                                                       skos:prefLabel "CellPolarisationPotential"@en .
```

Comment:
CellPolarizationPotential is included because it is a defined IEC term. It is not equivalent to any of the other terms because it implies non-equilibrium conditions, which distnguishes it from OpenCircuitVoltage.